.travis.yml

```
@@ -1,10 +1,29 @@
 git:
     quiet: true
+    depth: 1
 
 language: c
 
 compiler:
     - clang
     - gcc
 
-script: make check
+os:
+    - linux
+    - osx
+
+before_install:
+    - $CC --version
+    - export MAKEJOBS="-j$(getconf _NPROCESSORS_ONLN)"
+    - export TRAVIS_COMMIT_LOG=`git log --format=fuller -1`
```

```suggestion
    - export TRAVIS_COMMIT_LOG=`git log --format=fuller -2`
```
To have some context would not hurt.